### PR TITLE
Feature 3 neue formular felder

### PR DESCRIPTION
**Pull request by crunchy-amarillo**

### CODE REVIEW

File: templates/apps/inbox.html [hunk 1]
```diff
@@ -1,48 +1,69 @@
...
+            </h4>
+            <h6 class="card-subtitle mb-2 text-muted">{{app.gender}} für {{app.country}}</h6>
+            <p class="card-text"><strong>Motivation</strong></p>
+            <p class="card-text">{{app.motivation|linebreaks }}</p>
+            <p class="card-text"><strong>Fähigkeiten</strong></p>
+            <p class="card-text">{{app.skills|linebreaks }}</p>
+            <p class="card-text"><strong>Ethik-Frage</strong></p>
+            <p class="card-text">{{app.ethical_dilemma|linebreaks }}</p>
+            {% if app.internet_profiles %}
+            <p class="card-text"><strong>Internet Profile</strong></p>
```
Comment (gnunicorn) [inline templates/apps/inbox.html:26]: any particular reason you removed the indentation within `{%if %}`-clauses? I think it is helps to figure out which blocks belong too it, don't you?

File: templates/apps/inbox.html [hunk 2]
```diff
@@ -57,5 +78,10 @@ <h6 class="card-subtitle mb-2 text-muted">{{app.gender}} für {{app.country}}</h
       }
     });
   });
+
```
Comment (gnunicorn) [inline templates/apps/inbox.html]: Warum so viele leere Zeilen am Ende?

**Review by gnunicorn — APPROVED** [review]
Just two minor things to discuss.

File: templates/apps/inbox.html (hunk 1 above)
Comment (crunchy-amarillo) [reply templates/apps/inbox.html:26]: no particular reason. just <ALT>+<UMSCH>+<F> within my ide to format code automatically. O.o
i fixt this. please have also a look at my additional commits. i added a missing validation to my code.

File: appntr/views.py [hunk 3]
```diff
@@ -274,6 +274,14 @@ class ApplicationForm(ModelForm):
     comm_rules = forms.BooleanField(required=True,
                     label=mark_safe('Ich habe die in der Satzung festgelegten <a href="https://bewegung.jetzt/wp-content/uploads/2017/05/AnhangzurSatzungVerhaltens-Kodex-vom29.April2017.pdf" target="_blank">Verhaltensregeln</a> und <a href="https://docs.google.com/document/d/1_6vpN3qkpGe7ef3lgBybkByH7WybKe9-GvVYQRjNxkY/edit?usp=sharing" target="_blank">die internen Kommunikationsregel</a> von DIB wahrgenommen und bin bereit mich daran zu halten.'))
 
+    def clean_dib_participation_details(self):
+        details = self.cleaned_data['dib_participation_details']
+        check = self.cleaned_data['dib_participation']
+
+        if check and not details:
+            raise ValidationError('Bitte teile uns mit, an welchen Aktionen du konkret mitgewirkt hast.')
```
Comment (gnunicorn) [inline appntr/views.py]: DiB "Du" immer groß ("Du" anstatt "du" :D )

**Review by gnunicorn — APPROVED** [review]

File: appntr/views.py (hunk 3 above)
Comment (crunchy-amarillo) [reply appntr/views.py]: Fixed

**Pull request merged**